### ISSUE TITLE
Updating this repository so that it's somehow usable again??

### ISSUE THREAD

**Issue by pablopareja** [bio4j/bio4j-neo4j#11]
Hey @eparejatobes + @laughedelic !

It's too bad that we somewhat abandoned this long time ago already... 

I think it'd be a good idea to put it up to date little by little _(of course giving a much higher priority to the rest of titan related stuff)_ (I could do it from time to time when I don't have any other urgent stuff to do)_

At least we should update it so that it compiles with the changes we made to the domain model, _otherwise it's absurd to use it in any way IMHO_...

What do you think :question: 

By the way @eparejatobes what's the state of [angulillos-neo4j](https://github.com/bio4j/angulillos-neo4j) ?

**Comment by eparejatobes** [comment]
well I think it only makes sense to resurrect this _if_ we are doing it as for Titan. And, on angulillos-neo4j, it should be pretty easy to finish; only the index part is a bit more involved, due to them being in a state of flux on the Neo4j side.

**Comment by pablopareja** [comment]
yeah, I never meant having something different _(apart from the things that **must** be different due to implementation details of the specific technology)_

**Comment by eparejatobes** [comment]
If someone volunteers for finishing [bio4/angulillos-neo4j](https://github.com/bio4j/angulillos-neo4j), I'd be happy to help

**Comment by pablopareja** [comment]
I could do it I guess... but I would need a decent amount of help :smiley:

**Comment by eparejatobes** [comment]
It's already started, it should be pretty easy
